server/application_api: TestStatusAPICombinedStatementsWithFullScans failed

server/application_api.TestStatusAPICombinedStatementsWithFullScans [failed](https://mesolite.cluster.engflow.com/invocations/default/9c3af81e-0304-4e07-bc64-8c6c797f9516?testReportRun=2&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvc2VydmVyL2FwcGxpY2F0aW9uX2FwaTphcHBsaWNhdGlvbl9hcGlfdGVzdA==) on master @ [971ba39ae57f213d10f07b1cadad5404361fa66f](https://github.com/cockroachdb/cockroach/commits/971ba39ae57f213d10f07b1cadad5404361fa66f):


```
=== RUN   TestStatusAPICombinedStatementsWithFullScans
    test_log_scope.go:172: test logs captured to: outputs.zip/logTestStatusAPICombinedStatementsWithFullScans2590260946
    test_log_scope.go:82: use -show-logs to present logs inline
    test_server_shim.go:168: automatically injected a shared process virtual cluster under test; see comment at top of test_server_shim.go for details.
    sql_stats_test.go:874: condition failed to evaluate within 3m45s: expected statement 'SELECT name FROM players WHERE age < _' not found in response: {"statements":[{"key":{"key_data":{"query":"SELECT avg(goals) FROM players","app":"TestCombinedStatementsWithFullScans","distSQL":true,"vec":false,"full_scan":true,"database":"football","plan_hash":0,"query_summary":"SELECT avg(goals) FROM players","transaction_fingerprint_id":0},"aggregated_ts":"2026-05-23T07:00:00Z","aggregation_interval":0},"id":806411059905526523,"stats":{"count":1,"first_attempt_count":1,"max_retries":0,"legacy_last_err":"","legacy_last_err_redacted":"","num_rows":{"mean":1,"squared_diffs":0},"idle_lat":{"mean":0,"squared_diffs":0},"parse_lat":{"mean":0.00003843,"squared_diffs":0},"plan_lat":{"mean":0.00136639,"squared_diffs":0},"run_lat":{"mean":0.005383069,"squared_diffs":0},"service_lat":{"mean":0.006813979,"squared_diffs":0},"overhead_lat":{"mean":0.00002609,"squared_diffs":0},"sensitive_info":{"last_err":"","most_recent_plan_description":{"name":""},"most_recent_plan_timestamp":"0001-01-01T00:00:00Z"},"bytes_read":{"mean":119,"squared_diffs":0},"rows_read":{"mean":2,"squared_diffs":0},"rows_written":{"mean":0,"squared_diffs":0},"exec_stats":{"count":1,"network_bytes":{"mean":192,"squared_diffs":0},"max_mem_usage":{"mean":51200,"squared_diffs":0},"contention_time":{"mean":0,"squared_diffs":0},"network_messages":{"mean":3,"squared_diffs":0},"max_disk_usage":{"mean":0,"squared_diffs":0},"cpu_sql_nanos":{"mean":4228629,"squared_diffs":0},"mvcc_iterator_stats":{"step_count":{"mean":2,"squared_diffs":0},"step_count_internal":{"mean":2,"squared_diffs":0},"seek_count":{"mean":1,"squared_diffs":0},"seek_count_internal":{"mean":1,"squared_diffs":0},"block_bytes":{"mean":0,"squared_diffs":0},"block_bytes_in_cache":{"mean":0,"squared_diffs":0},"key_bytes":{"mean":0,"squared_diffs":0},"value_bytes":{"mean":85,"squared_diffs":0},"point_count":{"mean":2,"squared_diffs":0},"points_covered_by_range_tombstones":{"mean":0,"squared_diffs":0},"range_key_count":{"mean":0,"squared_diffs":0},"range_key_contained_points":{"mean":0,"squared_diffs":0},"range_key_skipped_points":{"mean":0,"squared_diffs":0}},"admission_wait_time":{"mean":0,"squared_diffs":0}},"sql_type":"TypeDML","last_exec_timestamp":"2026-05-23T07:59:57.488414199Z","nodes":[1,3],"kv_node_ids":[1],"regions":["test"],"plan_gists":["AgHUAQIAEAAAAAwGAg=="],"indexes":["106@1"],"latency_info":{"min":0.006813979,"max":0.006813979,"p50":0,"p90":0,"p99":0},"last_error_code":"","failure_count":0,"used_follower_read":false,"generic_count":0,"stmt_hints_count":0,"kv_cpu_time_nanos":{"mean":573140,"squared_diffs":0},"canary_stats":{"count":0,"run_lat":{"mean":0,"squared_diffs":0},"plan_lat":{"mean":0,"squared_diffs":0}},"stable_stats":{"count":0,"run_lat":{"mean":0,"squared_diffs":0},"plan_lat":{"mean":0,"squared_diffs":0}}},"txn_fingerprint_ids":[0]},{"key":{"key_data":{"query":"GRANT  SYSTEM VIEWACTIVITY TO authentic_user_noadmin","app":"","distSQL":false,"vec":false,"full_scan":false,"database":"defaultdb","plan_hash":0,"query_summary":"GRANT  SYSTEM VIEWACTIVITY TO authentic_user_noadmin","transaction_fingerprint_id":0},"aggregated_ts":"2026-05-23T07:00:00Z","aggregation_interval":0},"id":870064593363012594,"stats":{"count":1,"first_attempt_count":1,"max_retries":0,"legacy_last_err":"","legacy_last_err_redacted":"","num_rows":{"mean":0,"squared_diffs":0},"idle_lat":{"mean":0,"squared_diffs":0},"parse_lat":{"mean":0.00002368,"squared_diffs":0},"plan_lat":{"mean":0.00073671,"squared_diffs":0},"run_lat":{"mean":0.765709395,"squared_diffs":0},"service_lat":{"mean":0.766488916,"squared_diffs":0},"overhead_lat":{"mean":0.000019131,"squared_diffs":0},"sensitive_info":{"last_err":"","most_recent_plan_description":{"name":""},"most_recent_plan_timestamp":"0001-01-01T00:00:00Z"},"bytes_read":{"mean":0,"squared_diffs":0},"rows_read":{"mean":0,"squared_diffs":0},"rows_written":{"mean":0,"squared_diffs":0},"exec_stats":{"count":1,"network_bytes":{"mean":0,"squared_diffs":0},"max_mem_usage":{"mean":10240,"squared_diffs":0},"contention_time":{"mean":0,"squared_diffs":0},"network_messages":{"mean":0,"squared_diffs":0},"max_disk_usage":{"mean":0,"squared_diffs":0},"cpu_sql_nanos":{"mean":8708204,"squared_diffs":0},"mvcc_iterator_stats":{"step_count":{"mean":0,"squared_diffs":0},"step_count_internal":{"mean":0,"squared_diffs":0},"seek_count":{"mean":0,"squared_diffs":0},"seek_count_internal":{"mean":0,"squared_diffs":0},"block_bytes":{"mean":0,"squared_diffs":0},"block_bytes_in_cache":{"mean":0,"squared_diffs":0},"key_bytes":{"mean":0,"squared_diffs":0},"value_bytes":{"mean":0,"squared_diffs":0},"point_count":{"mean":0,"squared_diffs":0},"points_covered_by_range_tombstones":{"mean":0,"squared_diffs":0},"range_key_count":{"mean":0,"squared_diffs":0},"range_key_contained_points":{"mean":0,"squared_diffs":0},"range_key_skipped_points":{"mean":0,"squared_diffs":0}},"admission_wait_time":{"mean":0,"squared_diffs":0}},"sql_type":"TypeDCL","last_exec_timestamp":"2026-05-23T07:59:55.529121158Z","nodes":[3],"regions":["test"],"plan_gists":["Ais="],"latency_info":{"min":0.766488916,"max":0.766488916,"p50":0,"p90":0,"p99":0},"last_error_code":"","failure_count":0,"used_follower_read":false,"generic_count":0,"stmt_hints_count":0,"kv_cpu_time_nanos":{"mean":0,"squared_diffs":0},"canary_stats":{"count":0,"run_lat":{"mean":0,"squared_diffs":0},"plan_lat":{"mean":0,"squared_diffs":0}},"stable_stats":{"count":0,"run_lat":{"mean":0,"squared_diffs":0},"plan_lat":{"mean":0,"squared_diffs":0}}},"txn_fingerprint_ids":[0]},{"key":{"key_data":{"query":"CREATE TABLE players (id INT8 PRIMARY KEY, name STRING, \"position\" STRING, age INT8, goals INT8)","app":"TestCombinedStatementsWithFullScans","distSQL":false,"vec":false,"full_scan":false,"database":"football","plan_hash":0,"query_summary":"CREATE TABLE players (id INT8 PRIMARY KEY, name STRING, \"position\" STRING, age INT8, goals INT8)","transaction_fingerprint_id":0},"aggregated_ts":"2026-05-23T07:00:00Z","aggregation_interval":0},"id":2063754836119936715,"stats":{"count":1,"first_attempt_count":1,"max_retries":0,"legacy_last_err":"","legacy_last_err_redacted":"","num_rows":{"mean":0,"squared_diffs":0},"idle_lat":{"mean":0,"squared_diffs":0},"parse_lat":{"mean":0.000218049,"squared_diffs":0},"plan_lat":{"mean":0.16694887,"squared_diffs":0},"run_lat":{"mean":0.154531843,"squared_diffs":0},"service_lat":{"mean":0.321720352,"squared_diffs":0},"overhead_lat":{"mean":0.00002159,"squared_diffs":0},"sensitive_info":{"last_err":"","most_recent_plan_description":{"name":""},"most_recent_plan_timestamp":"0001-01-01T00:00:00Z"},"bytes_read":{"mean":0,"squared_diffs":0},"rows_read":{"mean":0,"squared_diffs":0},"rows_written":{"mean":0,"squared_diffs":0},"exec_stats":{"count":1,"network_bytes":{"mean":0,"squared_diffs":0},"max_mem_usage":{"mean":10240,"squared_diffs":0},"contention_time":{"mean":0,"squared_diffs":0},"network_messages":{"mean":0,"squared_diffs":0},"max_disk_usage":{"mean":0,"squared_diffs":0},"cpu_sql_nanos":{"mean":12696157,"squared_diffs":0},"mvcc_iterator_stats":{"step_count":{"mean":0,"squared_diffs":0},"step_count_internal":{"mean":0,"squared_diffs":0},"seek_count":{"mean":0,"squared_diffs":0},"seek_count_internal":{"mean":0,"squared_diffs":0},"block_bytes":{"mean":0,"squared_diffs":0},"block_bytes_in_cache":{"mean":0,"squared_diffs":0},"key_bytes":{"mean":0,"squared_diffs":0},"value_bytes":{"mean":0,"squared_diffs":0},"point_count":{"mean":0,"squared_diffs":0},"points_covered_by_range_tombstones":{"mean":0,"squared_diffs":0},"range_key_count":{"mean":0,"squared_diffs":0},"range_key_contained_points":{"mean":0,"squared_diffs":0},"range_key_skipped_points":{"mean":0,"squared_diffs":0}},"admission_wait_time":{"mean":0,"squared_diffs":0}},"sql_type":"TypeDDL","last_exec_timestamp":"2026-05-23T07:59:56.985131529Z","nodes":[3],"regions":["test"],"plan_gists":["AiXSAQ=="],"latency_info":{"min":0.321720352,"max":0.321720352,"p50":0,"p90":0,"p99":0},"last_error_code":"","failure_count":0,"used_follower_read":false,"generic_count":0,"stmt_hints_count":0,"kv_cpu_time_nanos":{"mean":0,"squared_diffs":0},"canary_stats":{"count":0,"run_lat":{"mean":0,"squared_diffs":0},"plan_lat":{"mean":0,"squared_diffs":0}},"stable_stats":{"count":0,"run_lat":{"mean":0,"squared_diffs":0},"plan_lat":{"mean":0,"squared_diffs":0}}},"txn_fingerprint_ids":[0]},{"key":{"key_data":{"query":"CREATE DATABASE football","app":"TestCombinedStatementsWithFullScans","distSQL":false,"vec":false,"full_scan":false,"database":"defaultdb","plan_hash":0,"query_summary":"CREATE DATABASE football","transaction_fingerprint_id":0},"aggregated_ts":"2026-05-23T07:00:00Z","aggregation_interval":0},"id":2349301928046834544,"stats":{"count":1,"first_attempt_count":1,"max_retries":0,"legacy_last_err":"","legacy_last_err_redacted":"","num_rows":{"mean":0,"squared_diffs":0},"idle_lat":{"mean":0,"squared_diffs":0},"parse_lat":{"mean":0.00001553,"squared_diffs":0},"plan_lat":{"mean":0.559614175,"squared_diffs":0},"run_lat":{"mean":0.018022716,"squared_diffs":0},"service_lat":{"mean":0.577671681,"squared_diffs":0},"overhead_lat":{"mean":0.00001926,"squared_diffs":0},"sensitive_info":{"last_err":"","most_recent_plan_description":{"name":""},"most_recent_plan_timestamp":"0001-01-01T00:00:00Z"},"bytes_read":{"mean":0,"squared_diffs":0},"rows_read":{"mean":0,"squared_diffs":0},"rows_written":{"mean":0,"squared_diffs":0},"exec_stats":{"count":1,"network_bytes":{"mean":0,"squared_diffs":0},"max_mem_usage":{"mean":10240,"squared_diffs":0},"contention_time":{"mean":0,"squared_diffs":0},"network_messages":{"mean":0,"squared_diffs":0},"max_disk_usage":{"mean":0,"squared_diffs":0},"cpu_sql_nanos":{"mean":28868103,"squared_diffs":0},"mvcc_iterator_stats":{"step_count":{"mean":0,"squared_diffs":0},"step_count_internal":{"mean":0,"squared_diffs":0},"seek_count":{"mean":0,"squared_diffs":0},"seek_count_internal":{"mean":0,"squared_diffs":0},"block_bytes":{"mean":0,"squared_diffs":0},"block_bytes_in_cache":{"mean":0,"squared_diffs":0},"key_bytes":{"mean":0,"squared_diffs":0},"value_bytes":{"mean":0,"squared_diffs":0},"point_count":{"mean":0,"squared_diffs":0},"points_covered_by_range_tombstones":{"mean":0,"squared_diffs":0},"range_key_count":{"mean":0,"squared_diffs":0},"range_key_contained_points":{"mean":0,"squared_diffs":0},"range_key_skipped_points":{"mean":0,"squared_diffs":0}},"admission_wait_time":{"mean":0,"squared_diffs":0}},"sql_type":"TypeDDL","last_exec_timestamp":"2026-05-23T07:59:56.255939723Z","nodes":[3],"regions":["test"],"plan_gists":["Ais="],"latency_info":{"min":0.577671681,"max":0.577671681,"p50":0,"p90":0,"p99":0},"last_error_code":"","failure_count":0,"used_follower_read":false,"generic_count":0,"stmt_hints_count":0,"kv_cpu_time_nanos":{"mean":0,"squared_diffs":0},"canary_stats":{"count":0,"run_lat":{"mean":0,"squared_diffs":0},"plan_lat":{"mean":0,"squared_diffs":0}},"stable_stats":{"count":0,"run_lat":{"mean":0,"squared_diffs":0},"plan_lat":{"mean":0,"squared_diffs":0}}},"txn_fingerprint_ids":[0]},{"key":{"key_data":{"query":"SELECT name FROM players WHERE age \u003e= _","app":"TestCombinedStatementsWithFullScans","distSQL":true,"vec":false,"full_scan":true,"database":"football","plan_hash":0,"query_summary":"SELECT name FROM players","transaction_fingerprint_id":0},"aggregated_ts":"2026-05-23T07:00:00Z","aggregation_interval":0},"id":6087125859261613747,"stats":{"count":1,"first_attempt_count":1,"max_retries":0,"legacy_last_err":"","legacy_last_err_redacted":"","num_rows":{"mean":2,"squared_diffs":0},"idle_lat":{"mean":0,"squared_diffs":0},"parse_lat":{"mean":0.00004491,"squared_diffs":0},"plan_lat":{"mean":0.001876899,"squared_diffs":0},"run_lat":{"mean":0.004966619,"squared_diffs":0},"service_lat":{"mean":0.006910218,"squared_diffs":0},"overhead_lat":{"mean":0.00002179,"squared_diffs":0},"sensitive_info":{"last_err":"","most_recent_plan_description":{"name":""},"most_recent_plan_timestamp":"0001-01-01T00:00:00Z"},"bytes_read":{"mean":119,"squared_diffs":0},"rows_read":{"mean":2,"squared_diffs":0},"rows_written":{"mean":0,"squared_diffs":0},"exec_stats":{"count":1,"network_bytes":{"mean":376,"squared_diffs":0},"max_mem_usage":{"mean":40960,"squared_diffs":0},"contention_time":{"mean":0,"squared_diffs":0},"network_messages":{"mean":5,"squared_diffs":0},"max_disk_usage":{"mean":0,"squared_diffs":0},"cpu_sql_nanos":{"mean":4329008,"squared_diffs":0},"mvcc_iterator_stats":{"step_count":{"mean":2,"squared_diffs":0},"step_count_internal":{"mean":2,"squared_diffs":0},"seek_count":{"mean":1,"squared_diffs":0},"seek_count_internal":{"mean":1,"squared_diffs":0},"block_bytes":{"mean":0,"squared_diffs":0},"block_bytes_in_cache":{"mean":0,"squared_diffs":0},"key_bytes":{"mean":0,"squared_diffs":0},"value_bytes":{"mean":85,"squared_diffs":0},"point_count":{"mean":2,"squared_diffs":0},"points_covered_by_range_tombstones":{"mean":0,"squared_diffs":0},"range_key_count":{"mean":0,"squared_diffs":0},"range_key_contained_points":{"mean":0,"squared_diffs":0},"range_key_skipped_points":{"mean":0,"squared_diffs":0}},"admission_wait_time":{"mean":0,"squared_diffs":0}},"sql_type":"TypeDML","last_exec_timestamp":"2026-05-23T07:59:57.497381347Z","nodes":[1,3],"kv_node_ids":[1],"regions":["test"],"plan_gists":["AgHUAQIACgAAAAMFAgYC"],"indexes":["106@1"],"latency_info":{"min":0.006910218,"max":0.006910218,"p50":0,"p90":0,"p99":0},"last_error_code":"","failure_count":0,"used_follower_read":false,"generic_count":0,"stmt_hints_count":0,"kv_cpu_time_nanos":{"mean":488290,"squared_diffs":0},"canary_stats":{"count":0,"run_lat":{"mean":0,"squared_diffs":0},"plan_lat":{"mean":0,"squared_diffs":0}},"stable_stats":{"count":0,"run_lat":{"mean":0,"squared_diffs":0},"plan_lat":{"mean":0,"squared_diffs":0}}},"txn_fingerprint_ids":[0]},{"key":{"key_data":{"query":"SET database = football","app":"TestCombinedStatementsWithFullScans","distSQL":false,"vec":false,"full_scan":false,"database":"football","plan_hash":0,"query_summary":"SET database = football","transaction_fingerprint_id":0},"aggregated_ts":"2026-05-23T07:00:00Z","aggregation_interval":0},"id":6152965469479632015,"stats":{"count":1,"first_attempt_count":1,"max_retries":0,"legacy_last_err":"","legacy_last_err_redacted":"","num_rows":{"mean":0,"squared_diffs":0},"idle_lat":{"mean":0,"squared_diffs":0},"parse_lat":{"mean":0.00002462,"squared_diffs":0},"plan_lat":{"mean":0.00031737,"squared_diffs":0},"run_lat":{"mean":0.275877704,"squared_diffs":0},"service_lat":{"mean":0.276240214,"squared_diffs":0},"overhead_lat":{"mean":0.00002052,"squared_diffs":0},"sensitive_info":{"last_err":"","most_recent_plan_description":{"name":""},"most_recent_plan_timestamp":"0001-01-01T00:00:00Z"},"bytes_read":{"mean":0,"squared_diffs":0},"rows_read":{"mean":0,"squared_diffs":0},"rows_written":{"mean":0,"squared_diffs":0},"exec_stats":{"count":1,"network_bytes":{"mean":0,"squared_diffs":0},"max_mem_usage":{"mean":10240,"squared_diffs":0},"contention_time":{"mean":0,"squared_diffs":0},"network_messages":{"mean":0,"squared_diffs":0},"max_disk_usage":{"mean":0,"squared_diffs":0},"cpu_sql_nanos":{"mean":5715818,"squared_diffs":0},"mvcc_iterator_stats":{"step_count":{"mean":0,"squared_diffs":0},"step_count_internal":{"mean":0,"squared_diffs":0},"seek_count":{"mean":0,"squared_diffs":0},"seek_count_internal":{"mean":0,"squared_diffs":0},"block_bytes":{"mean":0,"squared_diffs":0},"block_bytes_in_cache":{"mean":0,"squared_diffs":0},"key_bytes":{"mean":0,"squared_diffs":0},"value_bytes":{"mean":0,"squared_diffs":0},"point_count":{"mean":0,"squared_diffs":0},"points_covered_by_range_tombstones":{"mean":0,"squared_diffs":0},"range_key_count":{"mean":0,"squared_diffs":0},"range_key_contained_points":{"mean":0,"squared_diffs":0},"range_key_skipped_points":{"mean":0,"squared_diffs":0}},"admission_wait_time":{"mean":0,"squared_diffs":0}},"sql_type":"TypeDML","last_exec_timestamp":"2026-05-23T07:59:56.533818517Z","nodes":[3],"regions":["test"],"plan_gists":["Ais="],"latency_info":{"min":0.276240214,"max":0.276240214,"p50":0,"p90":0,"p99":0},"last_error_code":"","failure_count":0,"used_follower_read":false,"generic_count":0,"stmt_hints_count":0,"kv_cpu_time_nanos":{"mean":0,"squared_diffs":0},"canary_stats":{"count":0,"run_lat":{"mean":0,"squared_diffs":0},"plan_lat":{"mean":0,"squared_diffs":0}},"stable_stats":{"count":0,"run_lat":{"mean":0,"squared_diffs":0},"plan_lat":{"mean":0,"squared_diffs":0}}},"txn_fingerprint_ids":[0]},{"key":{"key_data":{"query":"INSERT INTO players(id, name, \"position\", age, goals) VALUES (_, __more__), (__more__)","app":"TestCombinedStatementsWithFullScans","distSQL":false,"vec":false,"full_scan":false,"database":"football","plan_hash":0,"query_summary":"INSERT INTO players(id, name, \"posi...)","transaction_fingerprint_id":0},"aggregated_ts":"2026-05-23T07:00:00Z","aggregation_interval":0},"id":6550303620005812375,"stats":{"count":1,"first_attempt_count":1,"max_retries":0,"legacy_last_err":"","legacy_last_err_redacted":"","num_rows":{"mean":2,"squared_diffs":0},"idle_lat":{"mean":0,"squared_diffs":0},"parse_lat":{"mean":0.00003149,"squared_diffs":0},"plan_lat":{"mean":0.438303435,"squared_diffs":0},"run_lat":{"mean":0.055414738,"squared_diffs":0},"service_lat":{"mean":0.493767613,"squared_diffs":0},"overhead_lat":{"mean":0.00001795,"squared_diffs":0},"sensitive_info":{"last_err":"","most_recent_plan_description":{"name":""},"most_recent_plan_timestamp":"0001-01-01T00:00:00Z"},"bytes_read":{"mean":0,"squared_diffs":0},"rows_read":{"mean":0,"squared_diffs":0},"rows_written":{"mean":2,"squared_diffs":0},"exec_stats":{"count":1,"network_bytes":{"mean":0,"squared_diffs":0},"max_mem_usage":{"mean":10240,"squared_diffs":0},"contention_time":{"mean":0,"squared_diffs":0},"network_messages":{"mean":0,"squared_diffs":0},"max_disk_usage":{"mean":0,"squared_diffs":0},"cpu_sql_nanos":{"mean":6557559,"squared_diffs":0},"mvcc_iterator_stats":{"step_count":{"mean":0,"squared_diffs":0},"step_count_internal":{"mean":0,"squared_diffs":0},"seek_count":{"mean":0,"squared_diffs":0},"seek_count_internal":{"mean":0,"squared_diffs":0},"block_bytes":{"mean":0,"squared_diffs":0},"block_bytes_in_cache":{"mean":0,"squared_diffs":0},"key_bytes":{"mean":0,"squared_diffs":0},"value_bytes":{"mean":0,"squared_diffs":0},"point_count":{"mean":0,"squared_diffs":0},"points_covered_by_range_tombstones":{"mean":0,"squared_diffs":0},"range_key_count":{"mean":0,"squared_diffs":0},"range_key_contained_points":{"mean":0,"squared_diffs":0},"range_key_skipped_points":{"mean":0,"squared_diffs":0}},"admission_wait_time":{"mean":0,"squared_diffs":0}},"sql_type":"TypeDML","last_exec_timestamp":"2026-05-23T07:59:57.480073231Z","nodes":[3],"regions":["test"],"plan_gists":["AiAE1AEB"],"latency_info":{"min":0.493767613,"max":0.493767613,"p50":0,"p90":0,"p99":0},"last_error_code":"","failure_count":0,"used_follower_read":false,"generic_count":0,"stmt_hints_count":0,"kv_cpu_time_nanos":{"mean":686079,"squared_diffs":0},"canary_stats":{"count":0,"run_lat":{"mean":0,"squared_diffs":0},"plan_lat":{"mean":0,"squared_diffs":0}},"stable_stats":{"count":0,"run_lat":{"mean":0,"squared_diffs":0},"plan_lat":{"mean":0,"squared_diffs":0}}},"txn_fingerprint_ids":[0]},{"key":{"key_data":{"query":"SET application_name = _","app":"","distSQL":false,"vec":false,"full_scan":false,"database":"defaultdb","plan_hash":0,"query_summary":"SET application_name = _","transaction_fingerprint_id":0},"aggregated_ts":"2026-05-23T07:00:00Z","aggregation_interval":0},"id":12121160597728358152,"stats":{"count":1,"first_attempt_count":1,"max_retries":0,"legacy_last_err":"","legacy_last_err_redacted":"","num_rows":{"mean":0,"squared_diffs":0},"idle_lat":{"mean":0,"squared_diffs":0},"parse_lat":{"mean":0.00014752,"squared_diffs":0},"plan_lat":{"mean":0.000311869,"squared_diffs":0},"run_lat":{"mean":0.001139629,"squared_diffs":0},"service_lat":{"mean":0.001625099,"squared_diffs":0},"overhead_lat":{"mean":0.000026081,"squared_diffs":0},"sensitive_info":{"last_err":"","most_recent_plan_description":{"name":""},"most_recent_plan_timestamp":"0001-01-01T00:00:00Z"},"bytes_read":{"mean":0,"squared_diffs":0},"rows_read":{"mean":0,"squared_diffs":0},"rows_written":{"mean":0,"squared_diffs":0},"exec_stats":{"count":1,"network_bytes":{"mean":0,"squared_diffs":0},"max_mem_usage":{"mean":10240,"squared_diffs":0},"contention_time":{"mean":0,"squared_diffs":0},"network_messages":{"mean":0,"squared_diffs":0},"max_disk_usage":{"mean":0,"squared_diffs":0},"cpu_sql_nanos":{"mean":1459290,"squared_diffs":0},"mvcc_iterator_stats":{"step_count":{"mean":0,"squared_diffs":0},"step_count_internal":{"mean":0,"squared_diffs":0},"seek_count":{"mean":0,"squared_diffs":0},"seek_count_internal":{"mean":0,"squared_diffs":0},"block_bytes":{"mean":0,"squared_diffs":0},"block_bytes_in_cache":{"mean":0,"squared_diffs":0},"key_bytes":{"mean":0,"squared_diffs":0},"value_bytes":{"mean":0,"squared_diffs":0},"point_count":{"mean":0,"squared_diffs":0},"points_covered_by_range_tombstones":{"mean":0,"squared_diffs":0},"range_key_count":{"mean":0,"squared_diffs":0},"range_key_contained_points":{"mean":0,"squared_diffs":0},"range_key_skipped_points":{"mean":0,"squared_diffs":0}},"admission_wait_time":{"mean":0,"squared_diffs":0}},"sql_type":"TypeDML","last_exec_timestamp":"2026-05-23T07:59:55.532094047Z","nodes":[3],"regions":["test"],"plan_gists":["Ais="],"latency_info":{"min":0.001625099,"max":0.001625099,"p50":0,"p90":0,"p99":0},"last_error_code":"","failure_count":0,"used_follower_read":false,"generic_count":0,"stmt_hints_count":0,"kv_cpu_time_nanos":{"mean":0,"squared_diffs":0},"canary_stats":{"count":0,"run_lat":{"mean":0,"squared_diffs":0},"plan_lat":{"mean":0,"squared_diffs":0}},"stable_stats":{"count":0,"run_lat":{"mean":0,"squared_diffs":0},"plan_lat":{"mean":0,"squared_diffs":0}}},"txn_fingerprint_ids":[0]},{"key":{"key_data":{"query":"DROP INDEX IF EXISTS idx_age","app":"TestCombinedStatementsWithFullScans","distSQL":false,"vec":false,"full_scan":false,"database":"football","plan_hash":0,"query_summary":"DROP INDEX IF EXISTS idx_age","transaction_fingerprint_id":0},"aggregated_ts":"2026-05-23T07:00:00Z","aggregation_interval":0},"id":14192874762904297423,"stats":{"count":1,"first_attempt_count":1,"max_retries":0,"legacy_last_err":"","legacy_last_err_redacted":"","num_rows":{"mean":0,"squared_diffs":0},"idle_lat":{"mean":0,"squared_diffs":0},"parse_lat":{"mean":0.00011522,"squared_diffs":0},"plan_lat":{"mean":0.291473351,"squared_diffs":0},"run_lat":{"mean":0.00137781,"squared_diffs":0},"service_lat":{"mean":0.292995881,"squared_diffs":0},"overhead_lat":{"mean":0.0000295,"squared_diffs":0},"sensitive_info":{"last_err":"","most_recent_plan_description":{"name":""},"most_recent_plan_timestamp":"0001-01-01T00:00:00Z"},"bytes_read":{"mean":0,"squared_diffs":0},"rows_read":{"mean":0,"squared_diffs":0},"rows_written":{"mean":0,"squared_diffs":0},"exec_stats":{"count":1,"network_bytes":{"mean":0,"squared_diffs":0},"max_mem_usage":{"mean":10240,"squared_diffs":0},"contention_time":{"mean":0,"squared_diffs":0},"network_messages":{"mean":0,"squared_diffs":0},"max_disk_usage":{"mean":0,"squared_diffs":0},"cpu_sql_nanos":{"mean":12162617,"squared_diffs":0},"mvcc_iterator_stats":{"step_count":{"mean":0,"squared_diffs":0},"step_count_internal":{"mean":0,"squared_diffs":0},"seek_count":{"mean":0,"squared_diffs":0},"seek_count_internal":{"mean":0,"squared_diffs":0},"block_bytes":{"mean":0,"squared_diffs":0},"block_bytes_in_cache":{"mean":0,"squared_diffs":0},"key_bytes":{"mean":0,"squared_diffs":0},"value_bytes":{"mean":0,"squared_diffs":0},"point_count":{"mean":0,"squared_diffs":0},"points_covered_by_range_tombstones":{"mean":0,"squared_diffs":0},"range_key_count":{"mean":0,"squared_diffs":0},"range_key_contained_points":{"mean":0,"squared_diffs":0},"range_key_skipped_points":{"mean":0,"squared_diffs":0}},"admission_wait_time":{"mean":0,"squared_diffs":0}},"sql_type":"TypeDDL","last_exec_timestamp":"2026-05-23T07:59:58.787497872Z","nodes":[3],"regions":["test"],"plan_gists":["Ais="],"latency_info":{"min":0.292995881,"max":0.292995881,"p50":0,"p90":0,"p99":0},"last_error_code":"","failure_count":0,"used_follower_read":false,"generic_count":0,"stmt_hints_count":0,"kv_cpu_time_nanos":{"mean":0,"squared_diffs":0},"canary_stats":{"count":0,"run_lat":{"mean":0,"squared_diffs":0},"plan_lat":{"mean":0,"squared_diffs":0}},"stable_stats":{"count":0,"run_lat":{"mean":0,"squared_diffs":0},"plan_lat":{"mean":0,"squared_diffs":0}}},"txn_fingerprint_ids":[0]}],"last_reset":"2026-05-23T07:59:49.799145453Z","internal_app_name_prefix":"$ internal","transactions":[{"stats_data":{"statement_fingerprint_ids":[12121160597728358152],"app":"","stats":{"count":1,"max_retries":0,"num_rows":{"mean":0,"squared_diffs":0},"service_lat":{"mean":0.002254549,"squared_diffs":0},"retry_lat":{"mean":0,"squared_diffs":0},"commit_lat":{"mean":0.00014836,"squared_diffs":0},"idle_lat":{"mean":0,"squared_diffs":0},"bytes_read":{"mean":0,"squared_diffs":0},"rows_read":{"mean":0,"squared_diffs":0},"exec_stats":{"count":1,"network_bytes":{"mean":0,"squared_diffs":0},"max_mem_usage":{"mean":10240,"squared_diffs":0},"contention_time":{"mean":0,"squared_diffs":0},"network_messages":{"mean":0,"squared_diffs":0},"max_disk_usage":{"mean":0,"squared_diffs":0},"cpu_sql_nanos":{"mean":1459290,"squared_diffs":0},"mvcc_iterator_stats":{"step_count":{"mean":0,"squared_diffs":0},"step_count_internal":{"mean":0,"squared_diffs":0},"seek_count":{"mean":0,"squared_diffs":0},"seek_count_internal":{"mean":0,"squared_diffs":0},"block_bytes":{"mean":0,"squared_diffs":0},"block_bytes_in_cache":{"mean":0,"squared_diffs":0},"key_bytes":{"mean":0,"squared_diffs":0},"value_bytes":{"mean":0,"squared_diffs":0},"point_count":{"mean":0,"squared_diffs":0},"points_covered_by_range_tombstones":{"mean":0,"squared_diffs":0},"range_key_count":{"mean":0,"squared_diffs":0},"range_key_contained_points":{"mean":0,"squared_diffs":0},"range_key_skipped_points":{"mean":0,"squared_diffs":0}},"admission_wait_time":{"mean":0,"squared_diffs":0}},"rows_written":{"mean":0,"squared_diffs":0},"kv_cpu_time_nanos":{"mean":0,"squared_diffs":0}},"aggregated_ts":"2026-05-23T07:00:00Z","transaction_fingerprint_id":528256080741849303,"aggregation_interval":0}},{"stats_data":{"statement_fingerprint_ids":[14192874762904297423],"app":"TestCombinedStatementsWithFullScans","stats":{"count":1,"max_retries":0,"num_rows":{"mean":0,"squared_diffs":0},"service_lat":{"mean":0.293823721,"squared_diffs":0},"retry_lat":{"mean":0,"squared_diffs":0},"commit_lat":{"mean":0.00019295,"squared_diffs":0},"idle_lat":{"mean":0,"squared_diffs":0},"bytes_read":{"mean":0,"squared_diffs":0},"rows_read":{"mean":0,"squared_diffs":0},"exec_stats":{"count":1,"network_bytes":{"mean":0,"squared_diffs":0},"max_mem_usage":{"mean":10240,"squared_diffs":0},"contention_time":{"mean":0,"squared_diffs":0},"network_messages":{"mean":0,"squared_diffs":0},"max_disk_usage":{"mean":0,"squared_diffs":0},"cpu_sql_nanos":{"mean":12162617,"squared_diffs":0},"mvcc_iterator_stats":{"step_count":{"mean":0,"squared_diffs":0},"step_count_internal":{"mean":0,"squared_diffs":0},"seek_count":{"mean":0,"squared_diffs":0},"seek_count_internal":{"mean":0,"squared_diffs":0},"block_bytes":{"mean":0,"squared_diffs":0},"block_bytes_in_cache":{"mean":0,"squared_diffs":0},"key_bytes":{"mean":0,"squared_diffs":0},"value_bytes":{"mean":0,"squared_diffs":0},"point_count":{"mean":0,"squared_diffs":0},"points_covered_by_range_tombstones":{"mean":0,"squared_diffs":0},"range_key_count":{"mean":0,"squared_diffs":0},"range_key_contained_points":{"mean":0,"squared_diffs":0},"range_key_skipped_points":{"mean":0,"squared_diffs":0}},"admission_wait_time":{"mean":0,"squared_diffs":0}},"rows_written":{"mean":0,"squared_diffs":0},"kv_cpu_time_nanos":{"mean":0,"squared_diffs":0}},"aggregated_ts":"2026-05-23T07:00:00Z","transaction_fingerprint_id":7751967277779904528,"aggregation_interval":0}},{"stats_data":{"statement_fingerprint_ids":[2349301928046834544],"app":"TestCombinedStatementsWithFullScans","stats":{"count":1,"max_retries":0,"num_rows":{"mean":0,"squared_diffs":0},"service_lat":{"mean":0.723456826,"squared_diffs":0},"retry_lat":{"mean":0,"squared_diffs":0},"commit_lat":{"mean":0.143904426,"squared_diffs":0},"idle_lat":{"mean":0,"squared_diffs":0},"bytes_read":{"mean":0,"squared_diffs":0},"rows_read":{"mean":0,"squared_diffs":0},"exec_stats":{"count":1,"network_bytes":{"mean":0,"squared_diffs":0},"max_mem_usage":{"mean":10240,"squared_diffs":0},"contention_time":{"mean":0,"squared_diffs":0},"network_messages":{"mean":0,"squared_diffs":0},"max_disk_usage":{"mean":0,"squared_diffs":0},"cpu_sql_nanos":{"mean":28868103,"squared_diffs":0},"mvcc_iterator_stats":{"step_count":{"mean":0,"squared_diffs":0},"step_count_internal":{"mean":0,"squared_diffs":0},"seek_count":{"mean":0,"squared_diffs":0},"seek_count_internal":{"mean":0,"squared_diffs":0},"block_bytes":{"mean":0,"squared_diffs":0},"block_bytes_in_cache":{"mean":0,"squared_diffs":0},"key_bytes":{"mean":0,"squared_diffs":0},"value_bytes":{"mean":0,"squared_diffs":0},"point_count":{"mean":0,"squared_diffs":0},"points_covered_by_range_tombstones":{"mean":0,"squared_diffs":0},"range_key_count":{"mean":0,"squared_diffs":0},"range_key_contained_points":{"mean":0,"squared_diffs":0},"range_key_skipped_points":{"mean":0,"squared_diffs":0}},"admission_wait_time":{"mean":0,"squared_diffs":0}},"rows_written":{"mean":0,"squared_diffs":0},"kv_cpu_time_nanos":{"mean":0,"squared_diffs":0}},"aggregated_ts":"2026-05-23T07:00:00Z","transaction_fingerprint_id":10374561692760573103,"aggregation_interval":0}},{"stats_data":{"statement_fingerprint_ids":[870064593363012594],"app":"","stats":{"count":1,"max_retries":0,"num_rows":{"mean":0,"squared_diffs":0},"service_lat":{"mean":1.32724244,"squared_diffs":0},"retry_lat":{"mean":0,"squared_diffs":0},"commit_lat":{"mean":0.066480094,"squared_diffs":0},"idle_lat":{"mean":0,"squared_diffs":0},"bytes_read":{"mean":0,"squared_diffs":0},"rows_read":{"mean":0,"squared_diffs":0},"exec_stats":{"count":1,"network_bytes":{"mean":0,"squared_diffs":0},"max_mem_usage":{"mean":10240,"squared_diffs":0},"contention_time":{"mean":0,"squared_diffs":0},"network_messages":{"mean":0,"squared_diffs":0},"max_disk_usage":{"mean":0,"squared_diffs":0},"cpu_sql_nanos":{"mean":8708204,"squared_diffs":0},"mvcc_iterator_stats":{"step_count":{"mean":0,"squared_diffs":0},"step_count_internal":{"mean":0,"squared_diffs":0},"seek_count":{"mean":0,"squared_diffs":0},"seek_count_internal":{"mean":0,"squared_diffs":0},"block_bytes":{"mean":0,"squared_diffs":0},"block_bytes_in_cache":{"mean":0,"squared_diffs":0},"key_bytes":{"mean":0,"squared_diffs":0},"value_bytes":{"mean":0,"squared_diffs":0},"point_count":{"mean":0,"squared_diffs":0},"points_covered_by_range_tombstones":{"mean":0,"squared_diffs":0},"range_key_count":{"mean":0,"squared_diffs":0},"range_key_contained_points":{"mean":0,"squared_diffs":0},"range_key_skipped_points":{"mean":0,"squared_diffs":0}},"admission_wait_time":{"mean":0,"squared_diffs":0}},"rows_written":{"mean":0,"squared_diffs":0},"kv_cpu_time_nanos":{"mean":0,"squared_diffs":0}},"aggregated_ts":"2026-05-23T07:00:00Z","transaction_fingerprint_id":11777100422732168237,"aggregation_interval":0}},{"stats_data":{"statement_fingerprint_ids":[806411059905526523],"app":"TestCombinedStatementsWithFullScans","stats":{"count":1,"max_retries":0,"num_rows":{"mean":1,"squared_diffs":0},"service_lat":{"mean":0.007854998,"squared_diffs":0},"retry_lat":{"mean":0,"squared_diffs":0},"commit_lat":{"mean":0.00023218,"squared_diffs":0},"idle_lat":{"mean":0,"squared_diffs":0},"bytes_read":{"mean":119,"squared_diffs":0},"rows_read":{"mean":2,"squared_diffs":0},"exec_stats":{"count":1,"network_bytes":{"mean":192,"squared_diffs":0},"max_mem_usage":{"mean":51200,"squared_diffs":0},"contention_time":{"mean":0,"squared_diffs":0},"network_messages":{"mean":3,"squared_diffs":0},"max_disk_usage":{"mean":0,"squared_diffs":0},"cpu_sql_nanos":{"mean":4228629,"squared_diffs":0},"mvcc_iterator_stats":{"step_count":{"mean":2,"squared_diffs":0},"step_count_internal":{"mean":2,"squared_diffs":0},"seek_count":{"mean":1,"squared_diffs":0},"seek_count_internal":{"mean":1,"squared_diffs":0},"block_bytes":{"mean":0,"squared_diffs":0},"block_bytes_in_cache":{"mean":0,"squared_diffs":0},"key_bytes":{"mean":0,"squared_diffs":0},"value_bytes":{"mean":85,"squared_diffs":0},"point_count":{"mean":2,"squared_diffs":0},"points_covered_by_range_tombstones":{"mean":0,"squared_diffs":0},"range_key_count":{"mean":0,"squared_diffs":0},"range_key_contained_points":{"mean":0,"squared_diffs":0},"range_key_skipped_points":{"mean":0,"squared_diffs":0}},"admission_wait_time":{"mean":0,"squared_diffs":0}},"rows_written":{"mean":0,"squared_diffs":0},"kv_cpu_time_nanos":{"mean":573140,"squared_diffs":0}},"aggregated_ts":"2026-05-23T07:00:00Z","transaction_fingerprint_id":11840895645803924772,"aggregation_interval":0}},{"stats_data":{"statement_fingerprint_ids":[2063754836119936715],"app":"TestCombinedStatementsWithFullScans","stats":{"count":1,"max_retries":0,"num_rows":{"mean":0,"squared_diffs":0},"service_lat":{"mean":0.450600683,"squared_diffs":0},"retry_lat":{"mean":0,"squared_diffs":0},"commit_lat":{"mean":0.060472435,"squared_diffs":0},"idle_lat":{"mean":0,"squared_diffs":0},"bytes_read":{"mean":0,"squared_diffs":0},"rows_read":{"mean":0,"squared_diffs":0},"exec_stats":{"count":1,"network_bytes":{"mean":0,"squared_diffs":0},"max_mem_usage":{"mean":10240,"squared_diffs":0},"contention_time":{"mean":0,"squared_diffs":0},"network_messages":{"mean":0,"squared_diffs":0},"max_disk_usage":{"mean":0,"squared_diffs":0},"cpu_sql_nanos":{"mean":12696157,"squared_diffs":0},"mvcc_iterator_stats":{"step_count":{"mean":0,"squared_diffs":0},"step_count_internal":{"mean":0,"squared_diffs":0},"seek_count":{"mean":0,"squared_diffs":0},"seek_count_internal":{"mean":0,"squared_diffs":0},"block_bytes":{"mean":0,"squared_diffs":0},"block_bytes_in_cache":{"mean":0,"squared_diffs":0},"key_bytes":{"mean":0,"squared_diffs":0},"value_bytes":{"mean":0,"squared_diffs":0},"point_count":{"mean":0,"squared_diffs":0},"points_covered_by_range_tombstones":{"mean":0,"squared_diffs":0},"range_key_count":{"mean":0,"squared_diffs":0},"range_key_contained_points":{"mean":0,"squared_diffs":0},"range_key_skipped_points":{"mean":0,"squared_diffs":0}},"admission_wait_time":{"mean":0,"squared_diffs":0}},"rows_written":{"mean":0,"squared_diffs":0},"kv_cpu_time_nanos":{"mean":0,"squared_diffs":0}},"aggregated_ts":"2026-05-23T07:00:00Z","transaction_fingerprint_id":12952444175679088916,"aggregation_interval":0}},{"stats_data":{"statement_fingerprint_ids":[6550303620005812375],"app":"TestCombinedStatementsWithFullScans","stats":{"count":1,"max_retries":0,"num_rows":{"mean":2,"squared_diffs":0},"service_lat":{"mean":0.494525892,"squared_diffs":0},"retry_lat":{"mean":0,"squared_diffs":0},"commit_lat":{"mean":0.060472435,"squared_diffs":0},"idle_lat":{"mean":0,"squared_diffs":0},"bytes_read":{"mean":0,"squared_diffs":0},"rows_read":{"mean":0,"squared_diffs":0},"exec_stats":{"count":1,"network_bytes":{"mean":0,"squared_diffs":0},"max_mem_usage":{"mean":10240,"squared_diffs":0},"contention_time":{"mean":0,"squared_diffs":0},"network_messages":{"mean":0,"squared_diffs":0},"max_disk_usage":{"mean":0,"squared_diffs":0},"cpu_sql_nanos":{"mean":6557559,"squared_diffs":0},"mvcc_iterator_stats":{"step_count":{"mean":0,"squared_diffs":0},"step_count_internal":{"mean":0,"squared_diffs":0},"seek_count":{"mean":0,"squared_diffs":0},"seek_count_internal":{"mean":0,"squared_diffs":0},"block_bytes":{"mean":0,"squared_diffs":0},"block_bytes_in_cache":{"mean":0,"squared_diffs":0},"key_bytes":{"mean":0,"squared_diffs":0},"value_bytes":{"mean":0,"squared_diffs":0},"point_count":{"mean":0,"squared_diffs":0},"points_covered_by_range_tombstones":{"mean":0,"squared_diffs":0},"range_key_count":{"mean":0,"squared_diffs":0},"range_key_contained_points":{"mean":0,"squared_diffs":0},"range_key_skipped_points":{"mean":0,"squared_diffs":0}},"admission_wait_time":{"mean":0,"squared_diffs":0}},"rows_written":{"mean":2,"squared_diffs":0},"kv_cpu_time_nanos":{"mean":686079,"squared_diffs":0}},"aggregated_ts":"2026-05-23T07:00:00Z","transaction_fingerprint_id":17691519375865507656,"aggregation_interval":0}},{"stats_data":{"statement_fingerprint_ids":[6152965469479632015],"app":"TestCombinedStatementsWithFullScans","stats":{"count":1,"max_retries":0,"num_rows":{"mean":0,"squared_diffs":0},"service_lat":{"mean":0.277347634,"squared_diffs":0},"retry_lat":{"mean":0,"squared_diffs":0},"commit_lat":{"mean":0.0001867,"squared_diffs":0},"idle_lat":{"mean":0,"squared_diffs":0},"bytes_read":{"mean":0,"squared_diffs":0},"rows_read":{"mean":0,"squared_diffs":0},"exec_stats":{"count":1,"network_bytes":{"mean":0,"squared_diffs":0},"max_mem_usage":{"mean":10240,"squared_diffs":0},"contention_time":{"mean":0,"squared_diffs":0},"network_messages":{"mean":0,"squared_diffs":0},"max_disk_usage":{"mean":0,"squared_diffs":0},"cpu_sql_nanos":{"mean":5715818,"squared_diffs":0},"mvcc_iterator_stats":{"step_count":{"mean":0,"squared_diffs":0},"step_count_internal":{"mean":0,"squared_diffs":0},"seek_count":{"mean":0,"squared_diffs":0},"seek_count_internal":{"mean":0,"squared_diffs":0},"block_bytes":{"mean":0,"squared_diffs":0},"block_bytes_in_cache":{"mean":0,"squared_diffs":0},"key_bytes":{"mean":0,"squared_diffs":0},"value_bytes":{"mean":0,"squared_diffs":0},"point_count":{"mean":0,"squared_diffs":0},"points_covered_by_range_tombstones":{"mean":0,"squared_diffs":0},"range_key_count":{"mean":0,"squared_diffs":0},"range_key_contained_points":{"mean":0,"squared_diffs":0},"range_key_skipped_points":{"mean":0,"squared_diffs":0}},"admission_wait_time":{"mean":0,"squared_diffs":0}},"rows_written":{"mean":0,"squared_diffs":0},"kv_cpu_time_nanos":{"mean":0,"squared_diffs":0}},"aggregated_ts":"2026-05-23T07:00:00Z","transaction_fingerprint_id":18014403122907204432,"aggregation_interval":0}},{"stats_data":{"statement_fingerprint_ids":[6087125859261613747],"app":"TestCombinedStatementsWithFullScans","stats":{"count":1,"max_retries":0,"num_rows":{"mean":2,"squared_diffs":0},"service_lat":{"mean":0.008386368,"squared_diffs":0},"retry_lat":{"mean":0,"squared_diffs":0},"commit_lat":{"mean":0.00014067,"squared_diffs":0},"idle_lat":{"mean":0,"squared_diffs":0},"bytes_read":{"mean":119,"squared_diffs":0},"rows_read":{"mean":2,"squared_diffs":0},"exec_stats":{"count":1,"network_bytes":{"mean":376,"squared_diffs":0},"max_mem_usage":{"mean":40960,"squared_diffs":0},"contention_time":{"mean":0,"squared_diffs":0},"network_messages":{"mean":5,"squared_diffs":0},"max_disk_usage":{"mean":0,"squared_diffs":0},"cpu_sql_nanos":{"mean":4329008,"squared_diffs":0},"mvcc_iterator_stats":{"step_count":{"mean":2,"squared_diffs":0},"step_count_internal":{"mean":2,"squared_diffs":0},"seek_count":{"mean":1,"squared_diffs":0},"seek_count_internal":{"mean":1,"squared_diffs":0},"block_bytes":{"mean":0,"squared_diffs":0},"block_bytes_in_cache":{"mean":0,"squared_diffs":0},"key_bytes":{"mean":0,"squared_diffs":0},"value_bytes":{"mean":85,"squared_diffs":0},"point_count":{"mean":2,"squared_diffs":0},"points_covered_by_range_tombstones":{"mean":0,"squared_diffs":0},"range_key_count":{"mean":0,"squared_diffs":0},"range_key_contained_points":{"mean":0,"squared_diffs":0},"range_key_skipped_points":{"mean":0,"squared_diffs":0}},"admission_wait_time":{"mean":0,"squared_diffs":0}},"rows_written":{"mean":0,"squared_diffs":0},"kv_cpu_time_nanos":{"mean":488290,"squared_diffs":0}},"aggregated_ts":"2026-05-23T07:00:00Z","transaction_fingerprint_id":18093895379167981932,"aggregation_interval":0}}],"stmts_total_runtime_secs":11.43469,"txns_total_runtime_secs":9.812728,"oldest_aggregated_ts_returned":"2026-05-23T07:00:00Z","stmts_source_table":"crdb_internal.statement_statistics","txns_source_table":"crdb_internal.transaction_statistics"}
        
        goroutine dump: /mnt/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/testlogs/pkg/server/application_api/application_api_test/shard_1_of_8_run_2_of_3/test.outputs/goroutine_dump_3618719108.txt
    panic.go:694: -- test log scope end --
test logs left over in: outputs.zip/logTestStatusAPICombinedStatementsWithFullScans2590260946
--- FAIL: TestStatusAPICombinedStatementsWithFullScans (255.37s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=2</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/obs-prs
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStatusAPICombinedStatementsWithFullScans.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-64194

--- a/pkg/server/application_api/sql_stats_test.go
+++ b/pkg/server/application_api/sql_stats_test.go
@@ -691,10 +691,15 @@ func TestStatusAPICombinedStatementsWithFullScans(t *testing.T) {
 	}
 	skip.UnderRace(t, "test is too slow to run under race")
 
+	settings := cluster.MakeTestingClusterSettings()
+	persistedsqlstats.SQLStatsFlushEnabled.Override(
+		context.Background(), &settings.SV, false,
+	)
 	statsKnobs := sqlstats.CreateTestingKnobs()
 	statsKnobs.SynchronousSQLStats = true
 	testCluster := serverutils.StartCluster(t, 3, base.TestClusterArgs{
 		ServerArgs: base.TestServerArgs{
+			Settings: settings,
 			Knobs: base.TestingKnobs{
 				SQLStatsKnobs: statsKnobs,
 				SpanConfig: &spanconfig.TestingKnobs{

